docs/run.rst

```diff
@@ -1,54 +1,66 @@
+.. _run:
+
 =============================
 Upload data and run pipelines
 =============================
 
-So far we were only inspecting data: accessing information and
+So far we were been inspecting data: accessing information and
 downloading files. Here we explain how to upload new data and run
 data analytics pipelines.
 
 The obvious way to start the analysis is to upload some data to Resolwe.
-We will do that by using process of type upload. It may sound strange at
-first, but uploading the data to Resolwe is exactly the same as running
-any other process. We have inputs - path to some file on our local
-computer, an algorithm (some file transfer protocol) and output -
-reference to a file freshly uploaded on Resolwe server.
+We do that with `upload processes`_. To upload the data
+we run the same commands as for running other analysis processes.
+We have inputs (path to some file on our local
+computer), an algorithm (to pre-process the input files) and
+output (a reference to the file that was just uploaded).
+
+.. _`upload processes`: http://resolwe-bio.readthedocs.io/en/latest/catalog.html#upload
 
 There are many upload processes, since there are many different types
-of files that can be uploaded. How to find all possible upload
-processes? Well, using the knowledge from previous chapter:
+of files we can upload. Where to get information about existing
+processes? You can browse the `Process catalog`_ or use the SDK:
+
+.. _`Process catalog`: http://resolwe-bio.readthedocs.io/en/latest/catalog.html
 
-.. literalinclude:: files/resdk-example.py
-   :lines: 63
+.. literalinclude:: files/example_run.py
+   :lines: 3-4
 
-Method run
-==========
+The run method
+==============
 
-Uploading file to Resolwe is as easy as:
+Uploading files to Resolwe is done with a single command. You can
+download example NGS read files here: `forward read pair`_,
+`reverse read pair`_.
 
-.. literalinclude:: files/resdk-example.py
-   :lines: 65-66
+.. _`forward read pair`: https://torta.bcm.genialis.com/data/553/reads_paired_abyss_1.fastq.gz?force_download=1
+.. _`reverse read pair`: https://torta.bcm.genialis.com/data/553/reads_paired_abyss_2.fastq.gz?force_download=1
+
+.. literalinclude:: files/example_run.py
+   :lines: 6-11
 
 As you can see, we have provided just process ``slug`` and ``input``
-parameters. What the method returns is a data object (in our case,
-called ``reads``). After uploading reads, we may want to assemble
-them with *Abyss*:
+parameters. What the method returns is a data object (here we call it
+``reads``). After the upload reads, we may want to assemble
```

Comment:
Not sure what was meant here. Choose:
"uploading the reads" or
"the reads are uploaded" or somthing similar.